.config/nextest.toml

```diff
@@ -1,8 +1,5 @@
 [profile.default]
 slow-timeout = { period = "60s", terminate-after = 2, grace-period = "0s" }
-# FIXME(nagisa): use --profile ci in CI instead when we manage to modify CI scripts...
-retries = { backoff = "fixed", count = 3, delay = "1s" }
-failure-output = "final"
```

Comment:
This change may be contentious. On one hand keeping this in makes flaky tests less of a major PiTA during local development. On the other we might end up living with the flaky tests in perpetuity if we keep this as nobody would have any motivation to actually look into these flakies…

Reply:
+1 to removing retries